docs/leap_sapi/sapi_rest.rst

```diff
@@ -163,7 +163,45 @@ resources and provides example usages.
 Examples Setup
 --------------
 
-The examples below use the following setup:
+To run the examples in this topic, set up your development environment, such as
+iPython, as described on the following tabs.
```

Comment:
it's `IPython`, but I'm not sure we need to call out one Python interpreter shell specifically.